integration/solana/calls.spec.ts

```diff
@@ -18,57 +18,58 @@ describe('Testing calls', function () {
             .accounts({ dataAccount: callee.storage.publicKey })
-            .rpc();
+            .rpc({commitment: "confirmed"});
 
         let res = await callee.program.methods.getX()
             .accounts({ dataAccount: callee.storage.publicKey })
-            .view();
+            .view({commitment: "confirmed"});
 
         expect(res).toEqual(new BN(102));
 
         res = await caller.program.methods.whoAmI()
-            .view();
+            .view({commitment: "confirmed"});
 
         expect(res).toStrictEqual(caller.program_key);
 
-        await caller.program.methods.doCall(callee.program_key, new BN(13123))
+        await caller.program.methods.doCall(new BN(13123))
             .accounts({
                 callee_dataAccount: callee.storage.publicKey,
-                callee_programId: callee.program_key,
+                callee_pid: callee.program_key,
                 })
-            .rpc();
+            .rpc({commitment: "confirmed"});
 
         res = await callee.program.methods.getX()
             .accounts({ dataAccount: callee.storage.publicKey })
-            .view();
+            .view({commitment: "confirmed"});
 
         expect(res).toEqual(new BN(13123));
 
-        res = await caller.program.methods.doCall2(callee.program_key, new BN(20000))
+        res = await caller.program.methods.doCall2(new BN(20000))
             .accounts({
                 callee_dataAccount: callee.storage.publicKey,
-                callee_programId: callee.program_key,
+                callee_pid: callee.program_key,
             })
-            .view();
+            .view({commitment: "confirmed"});
 
         expect(res).toEqual(new BN(33123));
 
-        res = await caller.program.methods.doCall3(callee.program_key, callee2.program_key, [new BN(3), new BN(5), new BN(7), new BN(9)], "yo")
+        res = await caller.program.methods.doCall3([new BN(3), new BN(5), new BN(7), new BN(9)], "yo")
             .accounts({
-                callee2_programId: callee2.program_key,
-                callee_programId: callee.program_key,
+                callee2_pid: callee2.program_key,
+                callee_pid: callee.program_key,
             })
-            .view();
+            .view({commitment: "confirmed"});
 
         expect(res.return0).toEqual(new BN(24));
         expect(res.return1).toBe("my name is callee");
 
-        res = await caller.program.methods.doCall4(callee.program_key, callee2.program_key, [new BN(1), new BN(2), new BN(3), new BN(4)], "asda")
+        res = await caller.program.methods.doCall4([new BN(1), new BN(2), new BN(3), new BN(4)], "asda")
             .accounts({
-                callee2_programId: callee2.program_key,
-                callee_programId: callee.program_key,
+                callee2_pid: callee2.program_key,
+                callee_pid: callee.program_key,
+                other_callee2: callee2.program_key,
             })
-            .view();
+            .view({commitment: "confirmed"});
```

Comment:
does this make any sense on view function?

Reply:
Without this, I get the `BlockHash not found` error locally.

Reply:
ah interesting :pray: